### PR TITLE
feat(str-rn-product): add locale to product feed

### PR DESCRIPTION
**Pull request by furkaneken**
Resolves STR-1328

### CODE REVIEW

File: react-native/storyly-react-native/RNStoryly.js [hunk 1]
```diff
@@ -207,6 +207,8 @@ class Storyly extends Component {
             storyHeaderShareIcon,
             storyFallbackIsEnabled,
             storyCartIsEnabled,
+            country,
```
Comment (levent-oral) [inline react-native/storyly-react-native/RNStoryly.js:210]: rename to storyProductCountry

File: react-native/storyly-react-native/RNStoryly.js [hunk 2]
```diff
@@ -207,6 +207,8 @@ class Storyly extends Component {
...
+            language,
```
Comment (levent-oral) [inline react-native/storyly-react-native/RNStoryly.js:211]: rename to storyProductLanguage

File: react-native/storyly-react-native/RNStoryly.js [hunk 3]
```diff
@@ -344,6 +348,8 @@ Storyly.propTypes = {
     storyGroupViewFactory: object,
     storyFallbackIsEnabled: bool,
     storyCartIsEnabled: bool,
+    country: string,
```
Comment (levent-oral) [inline react-native/storyly-react-native/RNStoryly.js:351]: rename to storyProductCountry

File: react-native/storyly-react-native/RNStoryly.js [hunk 4]
```diff
@@ -344,6 +348,8 @@ Storyly.propTypes = {
...
+    language: string,
```
Comment (levent-oral) [inline react-native/storyly-react-native/RNStoryly.js:352]: rename to storyProductLanguage

**Review by yigcal — APPROVED** [review]

**Pull request merged**